sw/device/silicon_creator/rom_ext/BUILD

```diff
@@ -142,3 +151,14 @@ opentitan_flash_binary(
         "//sw/device/silicon_creator/lib:manifest_def",
     ],
 )
+
+opentitan_flash_binary(
+    name = "pic",
```

Comment:
nit: we should probably call this "addr_translation" or "virtual_addr" so that it doesn't conflict with the actual PIC when we have it.